vignettes/dm.Rmd

```diff
@@ -4,7 +4,7 @@ date: "`r Sys.Date()`"
 output: rmarkdown::html_vignette
 vignette: >
   %\VignetteEncoding{UTF-8}
-  %\VignetteIndexEntry{First read: Getting started with dm}
```

Comment:
All changes to vignette index entry are made to avoid the following warning:

```r
#> Warning message:
#> The vignette title specified in \VignetteIndexEntry{} is different from the title in the YAML metadata.
```

Reply:
This was intentional so that the vignettes still look okay-ish on https://cran.r-project.org/package=dm .

Reply:
There is

```r
options(rmarkdown.html_vignette.check_title = FALSE)
```

Reply:
I mean "there is" in the sense that it could be used here

Reply:
I mean "there is" in the sense that it could be used here

Reply:
In that case, can we instead change the vignette _titles_ to be consistent with the vignette index entries on CRAN? 

That way, it will look good on CRAN, and there won't be any warnings.